tests/Unit/Evolution/Systems/GeneralizedHarmonic/BoundaryConditions/Bjorhus.py

```diff
@@ -14,30 +14,21 @@
 
 
```

Comment:
Can you format this file in a previous separate commit? I was shocked that removing a single arg form the c++ code could cause an almost 700 line change in a py file, until I realized it was largely unrelated formatting changes.